### PR TITLE
Allowing inputs on Sql.q (Query)

### PR DESCRIPTION
**Pull request by huttonr**
Adds support for inputs on Sql.q (queries).

Also fixes an error thrown when undefined or null is passed as opts.input and passes an empty object instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,19 +38,23 @@ Database connection settings are pulled from `Meteor.settings`, using the follow
 This allows a query to directly be run against the database. For SQL injection purposes,
 this should rarely be done. Better is a prepared statement, or stored procedure.
 
-#### Params: (query, callback)
+#### Params: (query : String, inputs : [ { name : 'param1', type : Sql.driver.TYPE, value : 'My Value' }, ... ])
```
Comment (emgee3) [inline README.md]: This changes the API in a way that is not backward compatible...

File: package.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: "emgee:mssql",
   summary: "mssql wrapper: non-reactive SQL Server package",
-  version: "1.3.0_3",
+  version: "1.3.0_4",
```
Comment (emgee3) [inline package.js]: But here the version is bumped by only a patch.

File: README.md [hunk 3]
```diff
@@ -38,19 +38,23 @@ Database connection settings are pulled from `Meteor.settings`, using the follow
...
```
Comment (emgee3) [inline README.md]: Not backward compatible...

**Comment by emgee3** [comment]
If we're going to make a backward incompatible change, the version number needs to reflect that. Seeing how this is really just a wrapper, perhaps it'd be smart to bump the mssql library as well, rather than break all the apps that currently use the library.

**Comment by huttonr** [comment]
The change made is actually backward-compatible as the callback was not actually implemented before.  I have now implemented it and `Sql.q(query, callback)` should now work.  e380990172ffe2eaa15caf39b9d7fc5e65514f92

**Pull request merged**